Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -175,8 +175,9 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	return size;
 }
 
-int GetVertexSize(int vtx_attr_group, bool preprocess)
+int GetVertexSize(int vtx_attr_group)
 {
+	bool preprocess = true;
```

Comment:
Why? I'm not even sure this is safe. Without deterministic dual core, this maybe won't get updated at all.